docs/2-developing/0-connectors/2-connector-specification.mdx

```diff
@@ -159,4 +159,18 @@ In this example, the default value in the generated specification will be
 `"https://example.com"`, i.e. if a user doesn't specify a URL, its value will be
 `"https://example.com"`.
 
+:::tip
+If your connector is on [v.12.0](https://github.com/ConduitIO/conduit-connector-sdk/releases/tag/v0.12.0) of the [Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk) and you want to migrate it to [v0.13.0](https://github.com/ConduitIO/conduit-connector-sdk/releases/tag/v0.13.0), we recommend you follow these steps:
```

Comment:
just because we had some bugs in 0.13.0
```suggestion
If your connector is on [v.12.0](https://github.com/ConduitIO/conduit-connector-sdk/releases/tag/v0.12.0) of the [Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk) and you want to migrate it to [v0.13](https://github.com/ConduitIO/conduit-connector-sdk/releases/tag/v0.13.1), we recommend you follow these steps:
```